.gitlint

```diff
@@ -0,0 +1,32 @@
+# See https://jorisroovers.com/gitlint/configuration for details
+[general]
+ignore=body-is-missing
+
+# Enable community contributed rules
+# See http://jorisroovers.github.io/gitlint/contrib_rules for details
+contrib=contrib-title-conventional-commits
+
+[title-max-length]
+line-length=72
+
+[title-min-length]
+min-length=10
+
+[title-match-regex]
+regex=: [^A-Z]
```

Comment:
Does this mean that the title should start with capital letter?

Reply:
No, there is actually a colon at the start of the regex so it only checks that after a colon (and a space) there isn't a capital letter. I copied it from our other repositories' config although in this case we could just check that everything is in lowercase as we don't use ticketing system here. But I think that would be a bit too strict.

Reply:
Oh right! The regex is a bit hard to read since it is not surrounded with quotes :)